src/Interface/Modules/Render/Screenshot.cc

```diff
@@ -49,11 +49,21 @@ Screenshot::Screenshot(QOpenGLWidget *glwidget, QObject *parent)
 
 void Screenshot::takeScreenshot()
 {
-  screenshot_ = viewport_->grabFramebuffer();
+	screenshot_ = getScreenshot();
 }
 
-QImage Screenshot::getScreenshot() {
-  return viewport_->grabFramebuffer();
+QImage Screenshot::getScreenshot()
+{
+  static constexpr int ALPHA_INDEX = 3;
+  static constexpr int OPAQUE = 255;
+	QImage image = viewport_->grabFramebuffer();
+	for (int j = 0; j < image.height(); ++j)
+	{
+    auto row = reinterpret_cast<QRgb*>(image.scanLine(j));
+		for (int i = 0; i < image.width(); ++i)
+			reinterpret_cast<uint8_t*>(row + i)[ALPHA_INDEX] = OPAQUE;
```

Comment:
```suggestion
			reinterpret_cast<uint8_t*>(row + i)[ALPHA_INDEX] = OPAQUE_PIXEL_VALUE;
```

Reply:
LOL @Haydelj You'll love this one: OPAQUE is #defined in a Windows header somewhere, so the build fails (and did you really test this on Windows??). So please change the identifier. @jessdtate will need to re-review.

Reply:
I suppose the Windows build was fine before the constexpr was pulled out. I really must get the github Windows build working...

Reply:
I wrote the fix on windows but I made your suggested style changes on Linux.

Reply:
Yeah cool. @ajanson confirmed it fixes Win7, so this should be good to go once the rename is in.